lib/jasmine-rails.rb

```diff
@@ -23,7 +23,7 @@ def route_path
 
     def spec_dir
       paths = jasmine_config['spec_dir'] || 'spec/javascripts'
-      [paths].flatten.map { |path| Dir.glob path }.flatten.collect { |path| Rails.root.join(path) }
+      [paths].flatten.map { |path| Dir.glob(Rails.root.join(path)) }.flatten.collect { |path| Rails.root.join(path) }
```

Comment:
```suggestion
      [paths].flatten.collect { |path| Pathname.glob Rails.root.join(path) }.flatten
```

That's how I fixed it, anyway. Seems a little more efficient and cleaner.